packages/cli/src/tools/packageManager.js

```diff
@@ -11,6 +11,32 @@ type Options = {|
 
 let projectDir;
 
+const yarnConfig = {
+  dependencies: ['add'],
+  devDependencies: ['add', '-D'],
+  uninstallDependencies: ['remove'],
```

Comment:
I'd prefer this shape and naming:
```js
const packageManagers = {
  yarn: {
  	add: ['add'],
  	addDev: ['add', '-D'],
  	remove: ['remove'],
  	install: ['install'],
  },
  npm: ...
}
```
It's gonna make it easier for us to add more (like `pnpm`) in the future